src/Select.js

```diff
@@ -0,0 +1,83 @@
+import React, { PropTypes, Component } from 'react';
+import cx from 'classnames';
+import constants from './constants';
+
+class Select extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { value: props.defaultValue };
+  }
+
+  componentDidMount() {
+    $(this.refs.__SELECT__).material_select();
+    $(this.refs.__SELECT__).on('change', this._onChange.bind(this));
+  }
+
+  componentDidUpdate() {
+    $(this.refs.__SELECT__).material_select();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({
+      value: nextProps.defaultValue
+    }, () => $(this.refs.__SELECT__).material_select());
+  }
+
+  componentWillUnmount() {
+    $(this.refs.__SELECT__).off('change', this._onChange.bind(this));
+  }
+
+  _onChange(e) {
+    this.setState({
+      value: e.target.value
+    }, () => this.props.onChange && this.props.onChange(e));
+  }
+
+  render() {
+    const { children, label, browserDefault } = this.props;
+    const classes = Object.assign(
```

Comment:
This is a nice way to create the size classes but it seems there's another way to do this without having to install yet another package, wdyt?